client/packages/common/src/hooks/useBreadcrumbs/useBreadcrumbs.tsx

```diff
@@ -47,7 +47,7 @@ export const useBreadcrumbs = (topLevelPaths: string[] = []) => {
     }, '');
     setUrlParts(urlParts);
     setSuffix(undefined);
-  }, [location]);
+  }, [pathname]);
```

Comment:
There is a react linter rule:
`"react-hooks/exhaustive-deps": "warn" // Checks effect dependencies`

https://reactjs.org/docs/hooks-rules.html

wounder if we should enable it, it usually quite easy to miss a dependency...

Reply:
I mostly find that one annoying 😁 Often there's a good reason ( I tell myself ) for not including a dependency which, strictly speaking, should be included. 
Maybe though..

Reply:
Thought the same when I first used it but then I was happy when it found some potential issues...